marimo/_messaging/ops.py

```diff
@@ -468,6 +469,14 @@ def __init__(
 
     def _stringify(self, value: object) -> str:
         try:
+            # HACK: We pretty-print tables to avoid str(ibis_table)
+            # which can be very slow when `ibis.options.interactive = True`
+            table_manager = get_table_manager_or_none(value)
+            if table_manager:
+                return str(table_manager)
+            else:
+                return str(value)[:50]
```

Comment:
this is an important change. and would cause long load times for large ibis tables